src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -457,7 +510,7 @@ public function thePromotionWasDisabledForTheChannel(PromotionInterface $promoti
     /**
      * @Given /^the (coupon "[^"]+") was used up to its usage limit$/
```

Comment:
Isn't this step redundant? We probably can use `(this coupon) has already reached its usage limit` implementation for that (https://github.com/Sylius/Sylius/pull/6273/files#diff-f5f3ee73d2030917831998b3bbd5cbb6R170)?